mscore/file.cpp

```diff
@@ -2311,6 +2311,7 @@ Score::FileError readScore(MasterScore* score, QString name, bool ignoreVersionE
                   { "gp4",  &importGTP                },
                   { "gp5",  &importGTP                },
                   { "gpx",  &importGTP                },
+                  { "gp",   &importGTP                },
                   { "ptb",  &importGTP                },
```

Comment:
Unrelated to this PR, but what is `ptb`? It seens PowerTaB, produced by Guitar Pro, TuxGuitar or Power Tab Editor? I don't see it mentioned anywhere else and the file selector doesn't allow for it. Shouldn't it?

Reply:
This is for PowerTab, yes. There is an importer for it in `mscore/importptb.cpp` which is called by `importGTP()` when the extension is ".ptb". If this importer works I think the extension should be allowed in the file selector.

Reply:
The question is whether this imported does work. Not really possible to test without having such PTB files.

Reply:
OTH without allowing for that extension we can't even test this, even if we had such PTB files